docs/dev/libvirt-howto.md

```diff
@@ -192,7 +192,12 @@ sudo virsh pool-autostart default
 ### Set up NetworkManager DNS overlay
 
 This step allows installer and users to resolve cluster-internal hostnames from your host.
-1. Edit `/etc/NetworkManager/NetworkManager.conf` and set `dns=dnsmasq` in section `[main]`
+1. Tell NetworkManager to use `dnsmasq`:
+
+```sh
```

Comment:
nit: this is part of the list entry, so the whole `sh` block should be indented four spaces.

Reply:
fixed